ionos_sbx/zammad/values.yaml

```diff
@@ -0,0 +1,15 @@
+zammad:
+  ingress:
+    enabled: true
+    className: nginx
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-sbx-issuer
```

Comment:
I would consider putting zammad iin ionos_common and in values-dev.yaml place only environemnt - specific values, i.e.

zammad:
  ingress:
    enabled: true
    className: nginx
    annotations:
      cert-manager.io/cluster-issuer: letsencrypt-dev-issuer
    hosts:
      - host: zammad.dome-marketplace-dev.org
        paths: 
          - path: /
            pathType: ImplementationSpecific
    tls:
      - secretName: zammad-tls-secret
        hosts:
          - zammad.dome-marketplace-dev.org

Reply:
yes I will definitely do it. but I would like to integrate the "ionos_common approach" in a dedicated PR (for all common components, not only zammad)